components/HamburgerMenu.vue

```diff
@@ -286,13 +308,21 @@ function logout() {
 }
 
 function setTheme(newTheme: string) {
-    document.documentElement.classList.remove('theme-orange', 'theme-coral', 'theme-violet', 'theme-ocean', 'theme-neon')
+    document.documentElement.classList.remove(
+        'theme-orange',
+        'theme-coral',
+        'theme-violet',
+        'theme-ocean',
+        'theme-neon',
+        'theme-accessible-high-contrast',
+        'theme-accessible-red-green'
+    )
     document.documentElement.classList.add(`theme-${newTheme}`)
     localStorage.setItem('theme', newTheme)
     currentTheme.value = newTheme
 }
 
-// this gives a black border to the selected theme in the hamburger menu
+// this gives a black border to the currently selected theme
```

Comment:
```suggestion
// This gives a black border to the currently selected theme
```

I know this comment was lowercase before you made changes, but it's good to fix it while we're at it.

Reply:
ah, bad habit, will fix this in upcoming commit.

Reply:
Fixed in upcoming commit.